src/components/Audioplayer/ScrollButton/index.js

```diff
@@ -1,49 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
-import Tooltip from 'react-bootstrap/lib/Tooltip';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 
-// eslint-disable-next-line no-confusing-arrow
-const scrollColor = props =>
-  props.active ? props.theme.brandPrimary : props.theme.textColor;
+import SwitchToggle from 'quran-components/lib/Toggle';
 
-const ScrollLink = styled.a`
-  width: 100%;
-  display: inline-block;
-  cursor: pointer;
-  padding-right: 1.5%;
-  color: ${scrollColor};
-  outline: none;
-  cursor: pointer;
-  margin-bottom: 0;
-`;
+import { ControlButton, StyledPopover } from '../index';
 
 const ScrollButton = ({ shouldScroll, onScrollToggle }) => {
   const tooltip = (
-    <Tooltip id="scroll-button-tooltip" placement="bottom">
-      <LocaleFormattedMessage
-        id="player.scrollButtonTip"
-        defaultMessage="Automatically scrolls to the currently playing ayah on transitions..."
+    <StyledPopover
+      id="scroll-toggle-popoverr"
+      title={
+        <LocaleFormattedMessage
+          id="player.scrollButtonTip"
+          defaultMessage="Automatically scrolls to the currently playing ayah on transitions..."
+        />
+      }
+    >
+      {'  '}
```

Comment:
better to style it instead of using empty spaces

Reply:
Toggle is already centered align. Forget to remove this :) thanks for reminding.